packages/python-kombu/python-kombu.spec

```diff
@@ -21,7 +21,6 @@ BuildRequires:  python2-devel
 BuildRequires:  python-setuptools
 BuildRequires:  python-nose
 
-BuildRequires:  python-nose-cover3
```

Comment:
Need a release bump?

Reply:
SInce this build succeeded upstream and is being released, I have not had a need to rebuild with this change, just propagating the change upstream.